Resources/Prototypes/Entities/Structures/Furniture/chairs.yml

```diff
@@ -107,6 +107,24 @@
     graph: Seat
     node: stoolBar
 
+- type: entity
+  name: brass chair
+  id: ChairBrass
+  parent: SeatBase
+  components:
+  - type: Transform
+    anchored: true
+  - type: Rotatable
+    rotateWhileAnchored: true
+  - type: Physics
+    bodyType: Static
+  - type: Anchorable
+  - type: Sprite
+    state: brass_chair
+  - type: Construction
+    graph: Seat
+    node: chairBrass
+
 - type: entity
```

Comment:
Just make a new parent and compose with that rather than copying 90% of a prototype.

Reply:
actually on closer observation it seems like doing that will mess with all the other chairs because they all are nearly the same and copy 90% of a prototype

Reply:
Dont ignore please.

it needs to be done, rather than making the problem even worse

Reply:
i'll make a separate PR for this

Reply:
done #25682

Reply:
made the brass chair consistent with the updated chair yaml 👍